pytalibstream/atr_keeper.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+import pyslidingwindow
+from pytalibstream.types.hlc import HLC
+
+
+class AtrKeeper:
+    def __init__(self, max_len: Optional[int] = 10):
```

Comment:
we should make this required.
don't want someone forget passing in the max_len when using this.